pmm/pmm2-ovf-staging-start.groovy

```diff
@@ -0,0 +1,106 @@
+import hudson.model.Node.Mode
```

Comment:
Can we leave a short comment at the beginning of each pipeline to understand its purpose. Question: will this stage a VM on DigitalOcean? Let's then mention it there.